installation/scripts/prow/ord-test.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+###
+# Following script installs necessary tooling for Debian, starts Director and Ord service, and runs the smoke tests.
+#
+
+set -o errexit
+
+function is_ready(){
+    local URL=${1}
+    local HTTP_CODE=$(curl -s -o /dev/null -I -w "%{http_code}" ${URL})
+    if [200 != "${HTTP_CODE}" ] ; then
+        echo "Response from  ${url} is still: ${HTTP_CODE}"
+        return 1
+    fi 
+    return 0
+}
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+INSTALLATION_DIR="$( cd "$( dirname "${CURRENT_DIR}/../../.." )" && pwd )"
+BASE_DIR="$( cd "$( dirname "${INSTALLATION_DIR}/../../../../../../.." )" && pwd )"
+JAVA_HOME="${BASE_DIR}/openjdk-11"
+M2_HOME="${BASE_DIR}/maven"
+COMPASS_DIR="$( cd "$( dirname "${INSTALLATION_DIR}/../.." )" && pwd )"  
+ORD_SVC_DIR="${BASE_DIR}/ord-service"
+
+mkdir -p "${JAVA_HOME}"
+export JAVA_HOME
+export PATH="${JAVA_HOME}/bin:${PATH}"
+
+mkdir -p "${M2_HOME}"
+export M2_HOME
+export PATH="${M2_HOME}/bin:${PATH}"
+
+export ARTIFACTS="/var/log/prow_artifacts"
+mkdir -p "${ARTIFACTS}"
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+
+    case ${key} in
+        --*)
+            echo "Unknown flag ${1}"
+            exit 1
+        ;;
+        *)    # unknown option
+            POSITIONAL+=("$1") # save it in an array for later
+            shift # past argument
+        ;;
+    esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+
+echo "Install java"
+curl -fLSs -o adoptopenjdk11.tgz "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.9.1%2B1/OpenJDK11U-jdk_x64_linux_hotspot_11.0.9.1_1.tar.gz"
+tar --extract --file adoptopenjdk11.tgz --directory "$JAVA_HOME" --strip-components 1 --no-same-owner
+rm adoptopenjdk11.tgz* 
+
+echo "Install maven"
+curl -fLSs -o apache-maven-3.8.5.tgz "https://dlcdn.apache.org/maven/maven-3/3.8.5/binaries/apache-maven-3.8.5-bin.tar.gz"
+tar --extract --file apache-maven-3.8.5.tgz --directory "$M2_HOME" --strip-components 1 --no-same-owner
+rm apache-maven-3.8.5.tgz* 
```

Comment:
We can add a flag to both director and ord-service's run.sh scripts  `--run-in-docker` which will build images from the Dockerfiles in the repos and run them in docker. 

This way the script will be executable from our machines too for local dev and will not rely on the OS of the VM in prow.

Reply:
If you provide both `--run-in-docker` and `--debug` we can return an error that debug is not supported in docker as the easiest solution for now. Later, we can expose the port of the debuger in the docker container when running it and use it in docker as well.